hail/python/hail/experimental/datasets.py

```diff
@@ -77,41 +79,42 @@ def load_dataset(name: str,
 
     names = set([dataset for dataset in datasets])
     if name not in names:
-        raise ValueError(f'{name} is not a dataset available in the'
-                         f' repository.')
+        raise ValueError(f'{name} is not a dataset available in the' f' repository.')
 
     versions = set(dataset['version'] for dataset in datasets[name]['versions'])
     if version not in versions:
-        raise ValueError(f'Version {repr(version)} not available for dataset'
-                         f' {repr(name)}.\n'
-                         f'Available versions: {versions}.')
+        raise ValueError(
+            f'Version {repr(version)} not available for dataset' f' {repr(name)}.\n' f'Available versions: {versions}.'
+        )
 
-    reference_genomes = set(dataset['reference_genome']
-                            for dataset in datasets[name]['versions'])
+    reference_genomes = set(dataset['reference_genome'] for dataset in datasets[name]['versions'])
     if reference_genome not in reference_genomes:
-        raise ValueError(f'Reference genome build {repr(reference_genome)} not'
-                         f' available for dataset {repr(name)}.\n'
-                         f'Available reference genome builds:'
-                         f' {reference_genomes}.')
-
-    clouds = set(k for dataset in datasets[name]['versions']
-                 for k in dataset['url'].keys())
+        raise ValueError(
+            f'Reference genome build {repr(reference_genome)} not'
+            f' available for dataset {repr(name)}.\n'
+            f'Available reference genome builds:'
+            f' {reference_genomes}.'
+        )
+
+    clouds = set(k for dataset in datasets[name]['versions'] for k in dataset['url'].keys())
     if cloud not in clouds:
-        raise ValueError(f'Cloud platform {repr(cloud)} not available for'
-                         f' dataset {name}.\n'
-                         f'Available platforms: {clouds}.')
+        raise ValueError(
+            f'Cloud platform {repr(cloud)} not available for' f' dataset {name}.\n' f'Available platforms: {clouds}.'
```

Comment:
This line is a little unfortunate. Let's combine them into one f-string.

Reply:
done!